apps/swirl-docs/src/components/CommandMenu/CustomHits.tsx

```diff
@@ -0,0 +1,117 @@
+import { SwirlIconDescription } from "@getflip/swirl-components-react";
```

Comment:
**suggestion:** This generates one tab stop per search result. Should we implement the menu pattern with arrow key controls?